test/fs/test_fs_js_api.c

```diff
@@ -118,12 +155,11 @@ int main() {
         FS.mkdir('/dir2');
     );
 
-    struct stat s;
-    stat("/dir1", &s);
-    assert(S_ISDIR(s.st_mode));
-    stat("/dir2", &s);
-    assert(S_ISDIR(s.st_mode));
-
+    struct stat rmdirStat;
+    stat("/dir1", &rmdirStat);
+    assert(S_ISDIR(rmdirStat.st_mode));
+    stat("/dir2", &rmdirStat);
+    assert(S_ISDIR(rmdirStat.st_mode));
```

Comment:
If the reason for the name change is to avoid a collision, another option is to add a scope,
```c
{
   struct stat s;
  ..
}
```

But the more explicit name also seems good by itself, either sgtm.

Or, it might be nice to split up the tests into their own functions, as the new test here.

Reply:
There was some collision after merging in the other tests, but I did move each test to its own function in the `truncate` PR after a suggestion from Sam, so the collision should be fixed (https://github.com/emscripten-core/emscripten/pull/19543). I was originally planning on changing the names back to `struct stat s` after the tests as individual functions are merged in.